README.md

```diff
@@ -165,11 +165,19 @@ flowchart LR
         click go-onewire "https://github.com/SpComb/go-onewire"
     end
 
+    subgraph "NETLINK_SOCK_DIAG"
+        direction LR
+
+        go-diag["github.com/florianl/go-diag"]
+        click go-onewire "https://github.com/florianl/go-diag"
```

Comment:
```suggestion
        click go-diag "https://github.com/florianl/go-diag"
```

Reply:
not sure, how I missed this :see_no_evil: